packages/ant-auth/prismerge.json

```diff
@@ -0,0 +1 @@
+{"core":{"inputs":["/home/hannes/projects/antify/antify/packages/ant-database/src/runtime/core/schema.prisma","/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/core/schema.prisma","/home/hannes/projects/antify/antify/packages/ant-auth/src/runtime/core/schema.prisma"],"fragments":["/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/core/schema.prisma"],"output":"/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/core/schema.merged.prisma"},"tenant":{"inputs":["/home/hannes/projects/antify/antify/packages/ant-database/src/runtime/tenant/schema.prisma","/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/tenant/schema.prisma"],"output":"/home/hannes/projects/antify/antify/packages/ant-auth/playground/server/datasources/tenant/schema.merged.prisma"}}
```

Comment:
This probably should not be commited, seems to me that it is generated and there are lokal pathes in here.